modules/troubleshooting-common-problems-with-distributed-workloads-for-administrators.adoc

```diff
@@ -0,0 +1,286 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-common-problems-with-distributed-workloads-for-administrators_{context}"]
+= Troubleshooting common problems with distributed workloads for administrators
+
+[role='_abstract']
+If your users are experiencing errors in {productname-long} relating to distributed workloads, read this section to understand what could be causing the problem, and how to resolve the problem.
+
+ifndef::upstream[]
+If you cannot see the problem here or in the release notes, contact {org-name} Support.
```

Comment:
Are these placeholders? - Will they be replaced?

Reply:
If you mean the text {productname-long} and {org-name} , these are attributes that are populated with the relevant value upstream and downstream :)

Reply:
That's brilliant!